packages/dataviews/src/style.scss

```diff
@@ -103,9 +106,18 @@
 			gap: $grid-unit-05;
 		}
 
+		th,
+		td {
+			&:first-child,
+			&:last-child {
+				transition: padding ease-out 0.1s;
+				@include reduce-motion("transition");
```

Comment:
Do you remember why this animation was added @jameskoster ? It's causing some weird effects when sorting / filtering...

Reply:
It was a suggestion by @jasmussen to avoid an instantaneous jump when the padding increased / decreased as you resize the viewport. 

Not essential imo.

Reply:
Yep, can remove if it causes issues.